src/templates/grading/response.html

```diff
@@ -79,6 +79,14 @@
           </div>
         </div>
       {% endfor %}<br>
+      
+      {% if supplementaryOrder %}  
+      <div id="supplementaryOrder">
+      {% for suppOrder in supplementaryOrder%}
+        {{suppOrder}}<br>
```

Comment:
Put some sort of text label here indicating what this info will be for.  In the screenshot you just have a grouping of IDs but its not really clear what the text is for.